packages/monitors/src/SyntheticPegMonitor.js

```diff
@@ -310,15 +310,14 @@ class SyntheticPegMonitor {
     // record the minimum and maximum.
     const latestTime = pricefeed.getLastUpdateTime();
 
+    // @dev: This might mean that the current price reported is a bit after the volatility window, but the error
+    // should be small enough that it shouldn't impact the results. Furthermore, the price is not used in the vol
+    // computation (which depends on the min/max), it only is reported alongside it as a reference point.
+    const pricefeedLatestPrice = await pricefeed.getCurrentPrice();
```

Comment:
Should we `try-catch` the `getCurrentPrice`?

Reply:
I think getCurrentPrice isn't supposed to error. According to the docs, it returns null if it doesn't have a price. Will toss a try-catch in just to be safe though. I also just realized it doesn't need an await.

Reply:
Fixed!